lessons/intro/numpy/index.ipynb

```diff
@@ -3949,20 +3949,6 @@
    "source": [
     "Což je podle [seznamu not](https://en.wikipedia.org/wiki/Piano_key_frequencies) skoro H$_2$ (123,5 Hz)."
    ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# Úkol"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Zadání úkolu je na Eduxu."
-   ]
```

Comment:
This works. Just a heads up: I'm planning to add actual links to the homework as I go through the course.